packages/osc-design-tokens/tokens/colors.js

```diff
@@ -109,6 +109,24 @@ for (const direction of directions) {
     gradients[`gradient-nonary${dirLabel}`] = `linear-gradient(${direction}, ${gradientLemonCyan})`;
 }
 
+// Bloom colours
+const bloomPrimary = 'hsl(56, 100%, 50%)';
+const bloomSecondary = 'hsl(187, 48%, 21%)';
+const bloomTertiary = 'hsl(42, 72%, 81%)';
+const bloomQuaternary = 'hsl(40, 56%, 95%)';
+const bloomQuinary = 'hsl(3, 71%, 57%)';
+const bloomSenary = 'hsl(33, 100%, 56%)';
+const bloomSeptanary = 'hsl(340, 77%, 90%)';
+const bloomOctonary = 'hsl(166, 39%, 65%)';
```

Comment:
Can you update these values to include the deg format, just so they're consistent with the other values
e.g
```suggestion
const bloomOctonary = 'hsl(166deg 39% 65%)';
```